package.json

```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "devStart": "nodemon server.js",
+    "devStart": "nodemon server/server.js",
+    "lint:fix": "eslint --fix ./server",
+    "prettier": "prettier ./server --write",
```

Comment:
add two more for just checking lint, and avoid fixing
so 
```
"lint" : "eslint  ./server"
```
similarly for other too